pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -87,11 +91,19 @@ let
     packpathDirs.myNeovimPackages = myVimPackage;
     finalPackdir = neovimUtils.packDir packpathDirs;
 
+    luaPluginRC = let
+      op = acc: normalizedPlugin:
+           acc ++ lib.optional (finalAttrs.autoconfigure && normalizedPlugin.plugin.passthru ? initLua) normalizedPlugin.plugin.passthru.initLua;
+      in
+        lib.foldl' op [] pluginsNormalized;
```

Comment:
question: `foldl'` will reverse the order (correct me if mistaken). Do we care about this?

Reply:
I dont think so, the accumulator comes first.
applying
```
diff --git a/pkgs/applications/editors/neovim/tests/default.nix b/pkgs/applications/editors/neovim/tests/default.nix
index fb89d47aceb0..48b00d42f272 100644
--- a/pkgs/applications/editors/neovim/tests/default.nix
+++ b/pkgs/applications/editors/neovim/tests/default.nix
@@ -170,7 +170,10 @@ in
   '';
 
   nvim_with_autoconfigure = pkgs.neovim.overrideAttrs(oa: {
-    plugins = [ vimPlugins.unicode-vim ];
+    plugins = [
+      vimPlugins.unicode-vim
+      vimPlugins.fzf-hoogle-vim
+    ];
     autoconfigure = true;
     # legacy wrapper sets it to false
     wrapRc = true;
```
generated 
```
vim.cmd.source "/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-init.vim"
vim.g.Unicode_data_directory="/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-vimplugin-unicode.vim-2024-07-23/autoload/unicode"
vim.g.hoogle_fzf_cache_file = vim.fn.stdpath('cache')..'/hoogle_cache.json
```